feat(performance-cleanup): homepage and trace view (#3354)
getsentry/sentry-docs: src/docs/product/performance/event-detail.mdx

---
title: "Event Detail"
sidebar_order: 6
description: "Learn more about Performance Event Detail and how you can drill all the way down into a span for a single transaction and traverse multiple directions. This will accelerate your ability to debug slow HTTP requests, database queries, identify associated errors, and root out other bottlenecks."
---

From [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can drill all the way down into a span for a single transaction and traverse multiple directions. This will accelerate your ability to debug slow HTTP requests, database queries, identify associated errors, and root out other bottlenecks.

> Entry points to an event detail are either through a [Transaction Summary](/product/performance/transaction-summary/) or [Query Builder](/product/discover-queries/query-builder/). In [Discover](/product/discover-queries/), performing a search with the `event.type:transaction` or going into a pre-built query such as "Transactions by Volume" will get you a list of event IDs. Event IDs will be linked to open the corresponding Event Detail.

Reviewer: ```suggestion

![Event detail showing critical event information, breakdowns, minimaps and waterfall view](./perf-transaction-detail.png)

You can get to an event detail from either a [Transaction Summary](/product/performance/transaction-summary/) or [Query Builder](/product/discover-queries/query-builder/). In [Discover](/product/discover-queries/), performing a search with the `event.type:transaction` or going into a pre-built query such as "Transactions by Volume" will generate a list of event IDs. Event IDs will be linked to open the corresponding **Event Detail** page.
```

<Alert>
Currently, only root transactions are searchable. Any span data that inherits from its root are not.
</Alert>

> ![Event detail showing critical event information, breakdowns, minimaps and waterfall view](./perf-transaction-detail.png)

Reviewer: ```suggestion
```

> Information about the specific event is located at the top, such as the Event ID, project, duration, time of occurrence, and status. More can be found under Tag Details.

Reviewer: ```suggestion
Information about the specific event, such as the event ID, project, duration, time of occurrence, and status, is located at the top. You can find more information under "Tag Details".
```

- **Children**: Events that the current event directly started
- **Descendants**: Future generations of events created by children of this event

![Quick trace showing related events](./quick-trace-open.png)

> A node will be in red with a flame if it is associated with errors. Click "Trace ID" to open the [Trace View](/product/performance/trace-view).

Reviewer: ```suggestion
If a node is associated with errors, it is displayed in red and has a flame icon next to it. Click "Trace ID" to open the [Trace View](/product/performance/trace-view).
```

## Span View

The span view is a split view where the left-hand side shows the transaction’s span tree, and the right-hand side represents each span as a colored rectangle. Within the tree view, Sentry identifies spans by their `op` and `description` values. If a span doesn’t have a description, Sentry uses the span’s id as a fallback. The first span listed is always the transaction’s root span, from which all other spans in the transaction descend.

![Span detail view shows the span id, trace id, parent span id, and other data such as tags.](./span-detail.png)

### Operations Breakdown

> This will correspond to the waterfall span view as a legend. The operation type, duration, and percentage of the total duration are shown.

Reviewer: ```suggestion
The operations breakdown corresponds to the waterfall span view as a legend. The operation type, duration, and percentage of the total duration are shown.
```
